omnibus/cookbooks/omnibus-supermarket/metadata.rb

```diff
@@ -9,6 +9,6 @@
 depends          'enterprise'
 depends          'omnibus'
 depends          'runit', '>= 3.0.3'
-depends          'sysctl'
+depends          'sysctl', '>= 0.10.0'
```

Comment:
Should this be `0.10.1` for [the "conf_dir not conf_file" fix](https://github.com/sous-chefs/sysctl/pull/94)?